src/wasm/html/index.html

```diff
@@ -0,0 +1,114 @@
+                            </div>
+                        </div>
+                    </div>
+                    <ul class="list-group list-group-flush pane" id="fileList">
+                        <li class="list-group-item text-center" id="emptyListInfo">
+                            <h5 class="card-title">The list is empty.</h5>
+                            <small>Add some files please.</small>
+                        </li>
+                        <template id="fileTemplate">
+                            <li class="list-group-item">
+                                <div class="d-flex w-100 justify-content-between">
+                                <div>
+                                    <input class="form-check-input me-1" type="radio" name="exeRadio" value="" id="firstRadio" checked>
+                                    <label class="form-check-label fileName" for="firstRadio">Some name</label>
+                                </div>
+                                <small>Some size</small>
+                                </div>
+                            </li>
+                        </template>
+                    </ul>
+                </div>
+                <div class="mt-2">
+                    <div class="collapse mb-2" id="collapseLogs">
+                        <h5>Logs</h5>
+                        <div id="con" class="card card-body"></div>
+                    </div>
+                    <p>
+                        <a class="btn btn-primary btn-sm" data-bs-toggle="collapse" href="#collapseLogs" role="button" aria-expanded="false" aria-controls="collapseLogs">
+                            <i class="bi bi-terminal-fill"></i> Show/hide logs
+                        </a>
+                    </p>
+                </div>
+            </div>
+            <div class="col-lg-6">
+                <h5>Output</h5>
+                <div class="card">
+                    <div class="card-header">
+                        <div class="d-flex w-100 justify-content-between">
+                            <div>
+                                <h5 id="title" class="card-title">Add and choose a EXE file...</h5>
+                                <h6 id="desc" class="card-subtitle mb-2 text-muted"></h6>
+                            </div>
+                            <small id="details">Size: <span id="size">0</span>MB Files: <span id="filesNum">0</span></small>
+                        </div>
+                        <div class="progress">
+                            <div id="progress-bar" class="progress-bar" role="progressbar" style="width: 0%;">0%</div>
+                        </div>
+                    </div>
+                    <div id="tree"></div> 
+                    <div class="card-footer">
+                        <button id="extractBtn" class="btn btn-warning w-100" disabled><i class="bi bi-file-earmark-zip-fill"></i> Extract and save as a ZIP! </button>
```

Comment:
It might be better to put this button somewhere near the top of the file list, maybe just below the progressbar.
If the users are not interested in selecting individual files, they shouldn't need to scroll down to the end of the list to click the extract button.